e2e/cephfs.go

```diff
@@ -329,6 +329,23 @@ var _ = Describe(cephfsType, func() {
 						framework.Failf("failed to delete CephFS storageclass: %v", err)
 					}
 				})
+
+				By("verify PVC with volumeBindingMode on helm installation", func() {
```

Comment:
What makes this special or unique for a Helm installation? This probably runs on non-Helm installations too? And, it should work on non-Helm installations as well.